zerver/lib/hotspots.py

```diff
@@ -89,6 +89,9 @@ def to_dict(self) -> Dict[str, str]:
     OneTimeNotice(
         name="intro_recent_view_modal",
     ),
+    OneTimeNotice(
+        name="first_stream_created_banner",
```

Comment:
Currently, It is actually a "modal" and not a "banner", so the naming is not quite correct.

Anyways, we [discussed in another PR](https://github.com/zulip/zulip/pull/29634#discussion_r1554050565) to make the names here independent of the suffix: "_modal" or "_banner", so even if we plan to change the way we display these notices, we won't have to do db migration at that time.